.github/workflows/commit_checker.yaml

```diff
@@ -24,5 +24,5 @@ jobs:
       - name: Check Commit Type
         uses: gsactions/commit-message-checker@v1
         with:
-          pattern: '^(Refs|Fixes) #YUPTOO-[0-9]+ - .+$'
-          error: 'Commit message should start with "Fixes #YUPTOO-<issue number> - "'
+          pattern: '^(Refs|Fixes) #RHINENG-[0-9]+ - .+$'
+          error: 'Commit message should start with "Fixes #RHINENG-<issue number> - "'
```

Comment:
I feel just `RHINENG-XXX` should be fine as a prefix rather than adding fixes or refs.
It is just my thought. Feel free to add your suggestions.

Reply:
I will modify the pattern that will accept without these prefix `Fixes/Refs` as well.